pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -58,23 +65,23 @@ func (p *criticalPaths) sort() {
 }
 
 func TestPreFilterState(t *testing.T) {
-	fooSelector := st.MakeLabelSelector().Exists("foo").Obj()
-	barSelector := st.MakeLabelSelector().Exists("bar").Obj()
+
 	tests := []struct {
-		name               string
-		pod                *v1.Pod
-		nodes              []*v1.Node
-		existingPods       []*v1.Pod
-		enableMinDomains   bool
-		objs               []runtime.Object
-		defaultConstraints []v1.TopologySpreadConstraint
-		want               *preFilterState
+		name                       string
+		pod                        *v1.Pod
+		nodes                      []*v1.Node
+		existingPods               []*v1.Pod
+		objs                       []runtime.Object
+		defaultConstraints         []v1.TopologySpreadConstraint
+		want                       *preFilterState
+		enableMinDomains           bool
+		enableNodeInclustionPolicy bool
```

Comment:
(not for this line)

Can you add cases to test a pod with multi-constraints (2 is enough), one passed NodeAffinityPolicy/NodeTaintsPolicy, while the other failed. We can add them in `TestPreFilterState()` and `TestMultipleConstraints()`.